community/neovim/depends

```diff
@@ -0,0 +1 @@
+cmake make
```

Comment:
Also missing a depend on `libtool`.

(`libtool make`)

Reply:
Also missing `automake make`.

Reply:
Addressed in https://github.com/kisslinux/community/pull/49/commits/2d67047abee64fd39d7a82ace7d3092356354c69